terraform/environments/electronic-monitoring-data/bastion_linux.tf

```diff
@@ -36,3 +36,56 @@ module "rds_bastion" {
   tags_common = local.tags
   tags_prefix = terraform.workspace
 }
+
+resource "aws_vpc_security_group_egress_rule" "access_ms_sql_server" {
+  security_group_id = module.rds_bastion.bastion_security_group
```

Comment:
`module.rds_bastion.bastion_security_group.id`?

Reply:
that is the id (not sure I agree with their naming scheme!) https://github.com/ministryofjustice/modernisation-platform-terraform-bastion-linux/blob/5784d4e3a3509f6e2d673849dc60dd0332a3dc7d/outputs.tf#L3